Testscripts/Windows/VERIFY-XDP-LOAD-UNLOAD.ps1

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+<#
+.Description
+    This script deploys the VM, verifies XDP with multiple load unload tests
+#>
+
+param([object] $AllVmData,
+    [object] $CurrentTestData)
+
+$MIN_KERNEL_VERSION = "5.6"
+
+# This function will start ping and xdpdump on client
+function Ping-XDPDump {
+    $VMData = $args[0]
+    $IP = $args[1]
+    $NIC = $args[2]
+    $LogSuffix = $args[3]
+    # Start Ping test
+    $ping_command = "ping -I $NIC -c 30 $IP > ~/pingOut$LogSuffix.txt"
+    Write-LogInfo "Starting command: $ping_command on $($VMData.RoleName)"
+    Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password `
+        -command $ping_command -RunInBackground -runAsSudo
+
+    # Start XDPDump
+    $xdp_command = "cd /root/bpf-samples/xdpdump && timeout 10 ./xdpdump -i $NIC > ~/xdpdumpout$LogSuffix.txt"
+    $testJob = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password `
+        -command $xdp_command -RunInBackground -runAsSudo
+    $timer = 0
+    while ((Get-Job -Id $testJob).State -eq "Running" ) {
+        $currentStatus = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password `
+            -command "tail -2 ~/xdpdumpout$LogSuffix.txt | head -1" -runAsSudo
+        Write-LogInfo "Current Test Status: $currentStatus"
+        Wait-Time -seconds 20
+        $timer += 1
+        if ($timer -gt 15) {
+            Throw "XDPSetup did not stop after 5 mins. Please check logs."

```

Comment:
throw

Reply:
Which log?

Reply:
what's the difference between throw and Throw? I tried and both are working same.